packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -288,6 +288,7 @@ module.exports = {
     // https://github.com/jmblog/how-to-optimize-momentjs-with-webpack
     // You can remove this if you don't use Moment.js:
     new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+    ...customConfig.webpackPlugins,
```

Comment:
dlls are obviously needed for prod builds - so include here.  Individual features can disable themselves via the `isDev` parameter.